pkg/backend/display/display.go

```diff
@@ -40,10 +40,10 @@ func printViewLive(out io.Writer, opts Options, permalink string) {
 	printPermalink(out, opts, "View Live", permalink)
 }
 
-// printOpenInBrowser print's an update's permalink prefaced with `Open in Browser (Ctrl+O): `. This is printed in
-// interactive scenarios that use the tree renderer.
-func printOpenInBrowser(term terminal.Terminal, opts Options, permalink string) {
```

Comment:
I'd almost name these `printPermalinkInteractive` and `printPermalinkNonInteractive`

Reply:
Renamed